### DISCUSSION TITLE
Fundme.sol functions not working

### DISCUSSION THREAD

**Discussion by DevinBeu** [Cyfrin/foundry-full-course-cu#2967] [General]
I depoloyed the fundme.sol contract to sepolia testnet. It went through, now when i try to call the fund function(red button), or either getVersion, or any of the blue buttons it just sais pending in the terminal and doesn't do anything. Its not reverting or finishing the transaction?
![testnet](https://github.com/user-attachments/assets/0a66edb7-4caf-41f4-9e28-14f0aaac38fc)

**Comment by chainflash** [comment]
The steps you took are correct  sometimes the issue is related to the remix. Clear your browser history and cookies then try again. It should work.

**Reply by DevinBeu** [reply]
i updated browser, cleared cache, and it worked.. Thanks!

**Reply by EngrPips** [reply]
Awesome

**Reply by chainflash** [reply]
greate
